src/components/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+// categoriesSlice.js
+/* eslint-disable */
+
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {
+    value: 'Update me',
+  },
+  reducers: {
+    checkStatus: (state) => {
+      state.value = 'Under construction...';
+    },
+  },
```

Comment:
- [ ] if the action.type is different, it should return the state with no changes, kindly add this to the reducer

Reply:
Implemented 👍